File: app/src/main/java/com/aircraft/toolmanagment/data/entity/Reminder.java
package com.aircraft.toolmanagment.data.entity;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "reminders")
public class Reminder {
    @PrimaryKey(autoGenerate = true)
    public int id;
    public int userId;
    public int toolId;
    public String reminderType;
    public String message;
    public long sentTime;

    public Reminder() {}
}

File: app/src/main/java/com/aircraft/toolmanagment/data/entity/Reminder.kt
package com.aircraft.toolmanagment.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "reminders")
data class Reminder(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val userId: Int = 0,
    val toolId: Int = 0,
    val reminderType: String = "",
    val message: String = "",
    val sentTime: Long = 0
)